rules/falco_rules.yaml

```diff
@@ -307,13 +305,13 @@
 # sshd, sendmail-msp, sendmail attempt to setuid to root even when running as non-root. Excluding here to avoid meaningless FPs
 - rule: non_sudo_setuid
   desc: an attempt to change users by calling setuid. sudo/su are excluded. user "root" is also excluded, as setuid calls typically involve dropping privileges.
-  condition: evt.type=setuid and evt.dir=> and not user.name=root and not userexec_binaries and not proc.name in (sshd, sendmail-msp, sendmail)
+  condition: evt.type=setuid and evt.dir=> and not user.name=root and not proc.name in (userexec_binaries, sshd, sendmail-msp, sendmail)
```

Comment:
Currently, lists cannot contain references to other lists. I had started on that support, but it made the code more complex and there wasn't a need for it with the current set of binaries.

Macros/rules, like this one, can contain a mix of individual items and list references, and you're right that the list reference is expanded directly to a single top-level set of items.